Makefile.am

```diff
@@ -158,14 +158,66 @@ base_files = \
 bin_PROGRAMS += csd
 
 csd_SOURCES = \
-	${base_files} \
+	${BASE_FILES} \
```

Comment:
I didn't get your point. The source files in $BASE_FILES are being built only once and when changed. In what cases will they be unnecessarily rebuilt?